test_framework/scripts/longhorn-setup.sh

```diff
@@ -56,9 +56,13 @@ install_cluster_autoscaler(){
 
 install_csi_snapshotter_crds(){
     CSI_SNAPSHOTTER_REPO_URL="https://github.com/kubernetes-csi/external-snapshotter.git"
-    CSI_SNAPSHOTTER_REPO_BRANCH="v6.2.1"
     CSI_SNAPSHOTTER_REPO_DIR="${TMPDIR}/k8s-csi-external-snapshotter"
 
+    [[ "${LONGHORN_REPO_URL}" =~ https://([^/]+)/([^/]+)/([^/.]+)(.git)? ]]
+    wget "https://raw.githubusercontent.com/${BASH_REMATCH[2]}/${BASH_REMATCH[3]}/${LONGHORN_REPO_BRANCH}/deploy/longhorn-images.txt" -O "/tmp/longhorn-images.txt"
+    IFS=: read -ra IMAGE_TAG_PAIR <<< $(grep csi-snapshotter /tmp/longhorn-images.txt)
+    CSI_SNAPSHOTTER_REPO_BRANCH="${IMAGE_TAG_PAIR[1]}"
```

Comment:
Could we have the CSI branch (version) visible in the Console Output?

Reply:
It can be found in console output:
```
+ install_csi_snapshotter_crds
+ CSI_SNAPSHOTTER_REPO_URL=https://github.com/kubernetes-csi/external-snapshotter.git
+ CSI_SNAPSHOTTER_REPO_DIR=/tmp/longhorn/k8s-csi-external-snapshotter
+ [[ https://github.com/longhorn/longhorn =~ https://([^/]+)/([^/]+)/([^/.]+)(.git)? ]]
+ wget https://raw.githubusercontent.com/longhorn/longhorn/v1.4.4/deploy/longhorn-images.txt -O /tmp/longhorn-images.txt
Connecting to raw.githubusercontent.com (185.199.110.133:443)

saving to '/tmp/longhorn-images.txt'
longhorn-images.txt  100% |********************************|   467  0:00:00 ETA
'/tmp/longhorn-images.txt' saved
+ IFS=:
+ read -ra IMAGE_TAG_PAIR
++ grep csi-snapshotter /tmp/longhorn-images.txt
+ CSI_SNAPSHOTTER_REPO_BRANCH=v5.0.1
+ git clone --single-branch --branch v5.0.1 https://github.com/kubernetes-csi/external-snapshotter.git /tmp/longhorn/k8s-csi-external-snapshotter
```